src/test/java/redis/clients/jedis/csc/UnifiedJedisClientSideCacheTestBase.java

```diff
@@ -94,7 +88,7 @@ public void cacheNotEmptyTest() {
       Cache cache = jedis.getCache();
       control.set("foo", "bar");
       assertEquals(0, cache.getSize());
-      assertEquals("bar", jedis.get("foo"));
+      AssertUtil.tryAssert(() -> jedis.get("foo"), "bar", 50, 100);
```

Comment:
Don't need here. This is the first command with the `jedis` client object. So it's always a cache miss -> a server call -> reply.